code/src/components/Todo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/Task'
+import styled from 'styled-components/macro'
+
+
+export const Todo = (props) => {
+
+  const dispatch = useDispatch()
+
+  const handleCheckboxClick = () => {
+    dispatch(tasks.actions.toggleComplete(props.item.id))
+  }
+
+  const handleRemoveButtonClick = () => {
+    dispatch(tasks.actions.removeItems(props.item.id))
+  }
+
+  return (
+    <div>
+      <List>
+        <label>
+          <Checkbox
+            type="checkbox"
+            arialabel="completed"
+            checked={props.item.complete}
+            onChange={handleCheckboxClick} />
+        </label>
+        {props.item.text}
```

Comment:
add this inside a `<p>` or another text tag.
Then to fix the 🗑to go to the right:
add `display: flex` to the ul, then add `flex-grow: 1` to the <p> with the todo text and then add a little `margin-right` to the trash bin button.